ert_shared/ensemble_evaluator/entity/ensemble_legacy.py

```
@@ -62,66 +62,82 @@ def _evaluate(self):
                 )
-            ]
-
-        queue_adaptor = QueueAdaptor(self._job_queue, self._config, self._ee_id)
-        futures.append(
-            queue_adaptor.execute_queue(
-                threading.BoundedSemaphore(value=CONCURRENT_INTERNALIZATION),
-                queue_evaluators,
             )
-        )
 
-        self._aggregate_future = asyncio.gather(*futures, return_exceptions=True)
-        self._allow_cancel.set()
-        try:
-            asyncio.get_event_loop().run_until_complete(self._aggregate_future)
-        except asyncio.CancelledError:
-            logger.debug("cancelled aggregate future")
-        else:
+            self._aggregate_future = asyncio.gather(*futures, return_exceptions=True)
+            self._allow_cancel.set()
+            try:
+                asyncio.get_event_loop().run_until_complete(self._aggregate_future)
+            except asyncio.CancelledError:
+                logger.debug("cancelled aggregate future")
+            else:
+                out_cloudevent = CloudEvent(
+                    {
+                        "type": identifiers.EVTYPE_ENSEMBLE_STOPPED,
+                        "source": f"/ert/ee/{self._ee_id}/ensemble",
+                        "id": str(uuid.uuid1()),
+                    }
+                )
+                asyncio.get_event_loop().run_until_complete(
+                    self.send_cloudevent(dispatch_url, out_cloudevent)
+                )
+        except Exception:
+            logger.exception(
+                "An exception occurred while starting the ensemble evaluation",
+                exc_info=True,
+            )
             out_cloudevent = CloudEvent(
                 {
-                    "type": identifiers.EVTYPE_ENSEMBLE_STOPPED,
+                    "type": identifiers.EVTYPE_ENSEMBLE_FAILED,
                     "source": f"/ert/ee/{self._ee_id}/ensemble",
                     "id": str(uuid.uuid1()),
                 }
```

Add `e` as data on the event? Sorry if I am repeating myself here, I can't remember what we concluded, but it's a great shame if that exception is lost.

---

See above, its logged, so its not entirely lost, the whole trace goes in the log.